.github/workflows/linux.yml

```diff
@@ -62,6 +62,42 @@ jobs:
           path:
             opensc*.tar.gz
 
+  build-arm64:
+    name: build-${{ matrix.dist }}-${{ matrix.arch }}
+    strategy:
+      matrix:
+        dist: ['22.04', '24.04']
+        arch: ['arm64']
+    runs-on: ubuntu-${{ matrix.dist }}${{ matrix.arch == 'arm64' && '-arm' || '' }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: awalsh128/cache-apt-pkgs-action@v1.4.3
+        with:
+          packages: ${{ env.BASE_DEPS }}
+          version: apt-${{ matrix.dist }}-${{ matrix.arch }}-base
+          execute_install_scripts: true
+      - run: .github/setup-linux.sh
+      - run: .github/build.sh dist
+      - name: Upload test logs
+        uses: actions/upload-artifact@v4
+        if: failure()
+        with:
+          name: ubuntu-${{ matrix.dist }}-${{ matrix.arch }}-test-logs
+          path: |
+            tests/*.log
+            src/tests/unittests/*.log
+      - uses: actions/cache@v4
+        id: cache-build
+        with:
+          path: ./*
+          key: ${{ runner.os }}-${{ matrix.dist }}-${{ matrix.arch }}-${{ github.sha }}
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: opensc-build-${{ matrix.dist }}-${{ matrix.arch }}
+          path:
+            opensc*.tar.gz
```

Comment:
Similarly here. This is done in the main dist job to publish the release tarball (which should be architecture independent anyway). So in theory, it might be used afterward to do some checks they really match, but there will be different timestamps anyway so I would probably drop this step too.

Reply:
Runs only on amd64 build